exercises/grade-school/grade_school_test.rb

```diff
@@ -4,85 +4,104 @@
 require_relative 'grade_school'
 
 class SchoolTest < Minitest::Test
-  attr_reader :school
-
-  def setup
-    @school = School.new
-  end
-
-  def test_an_empty_school
-    assert_equal({}, school.to_h)
+  def test_empty_grade
+    subject = School.new
```

Comment:
The most straight forward way in Ruby, yes.  I am not referring to Ruby, though, I am referring to the task at hand, from a "readme" point of view.  "Ordered list" from a mathematics view, from a programming view, or from a "Teacher in a school" view?  I most often see them on a piece of paper, as written list of names.  How they got that way, I don't concern myself, as a teacher, just that the report is correct.  This removes the worry of "how was it implemented" but when we test directly against implementation, it can make things more restricted.  The changes here are driven from "are we enforcing a single implementation" and so this is just me rambling with that in the back of my mind.

We have done it before, using to_s t show what is expected in other exercisms.  This frees up the implementation, because you are basing the test on a final report, rather than the specifics of a return in implementation.

The reference to a hash is being used later for students by grade, true.  But it doesn't have to be that. 

It is just me rambling, just reflecting on what I see.  Not that actions need to be taken on it.  It is likely good enough.